### DISCUSSION TITLE
How are people deploying their apps?

### DISCUSSION THREAD

**Discussion by NanoscaleSimulations** [reflex-dev/reflex#1317] [Show and tell]
Can we start a show and tell about how people are deploying their pynecone apps?
I am sure I am not the only one wanting to see examples of:
- what hosting services are you using?
- Dockerfiles
- Reverse proxy use
- DevOps pipelines
- GitHub actions
Etc.

I know this is something I keep having trouble with the most.

**Comment by FHU-yezi** [comment]
We do have a example Dockerfile, it is in '/docker-example'.

**Comment by NanoscaleSimulations** [comment]
It would be interesting to see other varieties of Dockerfiles for maybe having volumes

**Comment by DanShinde** [comment]
It would be really helpful if there were samples of Docker files for different hosting platforms. Many developers who try Python and want to get into web development have a hard time with DevOps.
